radio/src/pulses/pxx_arm.cpp

```diff
@@ -405,3 +422,22 @@ void setupPulsesPXX(uint8_t port)
 
   putPcmTail(port);
 }
+
+void setupPulsesPXX(uint8_t port)
+{
+  initPcmArray(port);
+
+#if defined(PXX_FREQUENCY_HIGH)
+  setupFramePXX(port, 0);
+  if (NUM_CHANNELS(port) > 8) {
+    setupFramePXX(port, 8);
```

Comment:
the array size needs to be checked carefully, theorically the size would need to be 2 times bigger

Reply:
checked, it is OK